docs/data/sdk-debuggability.md

```diff
@@ -0,0 +1,36 @@
+---
+title: SDK Debuggability 
+description: The SDK Debuggability document is a concise guide that provides solutions for common problems, debugging techniques, best practices, and troubleshooting guides to help developers efficiently resolve the SDKs related issues.
+---
+
+Data validation is a critical step in the instrumentation process. To streamline this process and enhance your debugging efforts, Amplitude provides some tools to convenience your debugging process. You can explore these in detail by following this [link](../debugger). These resources will facilitate the smooth implementation and operation of your projects.
+
+The following sections will outline common issues that you may encounter, along with their respective solutions or explanations to aid in resolving these problems. Please go to each SDK to check the specific debugbility doc. 
+
```

Comment:
I think we should have a section about "events not sending, events are delayed" and talk about the default queuing and flush behavior in the SDKs.

Reply:
I added this into the `Client Event Time`  section.